### PR TITLE
MES-6273 - e2e screens for catb

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

create placeholder screens for catb journey

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

https://user-images.githubusercontent.com/48550267/114850144-60f38180-9dd8-11eb-8d3c-68283e653916.mov

### CODE REVIEW

File: src/app/providers/route-by-category/__mocks__/route-by-category.mock.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export class RouteByCategoryProviderMock {
+  navigateToPage() {}
```
Comment (Cullima) [inline src/app/providers/route-by-category/__mocks__/route-by-category.mock.ts]: doesn't real implementation return a promise? maybe `navigateToPage = () => Promise.resolve()`

File: src/app/providers/route-by-category/route-by-category.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+@Injectable()
+export class RouteByCategoryProvider {
+  constructor(
+    public router: Router,
+  ) {
+  }
+
+  async navigateToPage(page: PageNameKeys, category?: TestCategory): Promise<void> {
+    const { config } = this.router;
+    const categoryPage = category ? getPageNameByCategoryAndKey(category, page) : page;
```
Comment (Cullima) [inline src/app/providers/route-by-category/route-by-category.ts]: can you add types to these constants

File: src/app/providers/route-by-category/route-by-category.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    config.push({
+      path: categoryPage,
+      loadChildren: () =>
+        import(`../../pages/${importPath}.module`)
+          .then((m) => m[`${categoryPage}Module`]),
+    });
+    await this.router.navigate([categoryPage]);
+  }
+
+  categoryToPage(category: TestCategory): string {
```
Comment (Cullima) [inline src/app/providers/route-by-category/route-by-category.ts]: think this name needs to be more descriptive of what it's doing e.g. `getPageAliasByCategory()` ?

**Review by matthew2564 — APPROVED** [review]

File: src/app/providers/route-by-category/route-by-category.ts [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      case TestCategory.K:
+      case TestCategory.H:
+      case TestCategory.G:
+      case TestCategory.F:
+        return 'cat-home-test';
+      default:
+    }
+  }
+
+  pagePath(page: string): string {
```
Comment (Cullima) [inline src/app/providers/route-by-category/route-by-category.ts]: again, can you make more descriptive, e.g. `getBasePagePathByCategory()` or something

**Review by Cullima — CHANGES_REQUESTED** [review]

File: src/app/providers/route-by-category/__mocks__/route-by-category.mock.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/app/providers/route-by-category/__mocks__/route-by-category.mock.ts]: sure

File: src/app/providers/route-by-category/route-by-category.ts (hunk 2 above)
Comment (andrewsetterfield) [reply src/app/providers/route-by-category/route-by-category.ts]: added

File: src/app/providers/route-by-category/route-by-category.ts (hunk 3 above)
Comment (andrewsetterfield) [reply src/app/providers/route-by-category/route-by-category.ts]: function name changed

File: src/app/providers/route-by-category/route-by-category.ts (hunk 4 above)
Comment (andrewsetterfield) [reply src/app/providers/route-by-category/route-by-category.ts]: function name changed

**Review by Cullima — APPROVED** [review]

**Pull request merged**